docker-compose.yaml

```diff
@@ -1,14 +1,36 @@
 version: "3.9"
 
-services:
+volumes:
+  db:
 
-  rails_api:
-    container_name: rails_api
-    build: .
+services:
+  web:
+    build:
+      target: web
+      context: .
+      dockerfile: Dockerfile
     volumes:
       - "./:/app/"
       - "./log:/app/log"
+    env_file:
+      - .env
+      - .env.example
```

Comment:
I don't like this pulling from the .env.example but for now since your repo doesn't commit the .env for dev, it has both.

Reply:
Why do we need `.env.example` here, can we add env vars in CI runtime? Or we can commit `test.env` instead of using `.env.example` if it's for running tests(no tests written so far btw).